subprojects/docs/src/docs/userguide/inspectingDependencies.adoc

```diff
@@ -84,3 +84,20 @@ Every Gradle project provides the task `dependencyInsight` to render the so-call
     <output args="-q dependencyInsight --dependency commons-codec --configuration scm"/>
 </sample>
 ++++
+
+[[sec:dependency_declaration_reasons]]
+=== Justifying dependency declarations with custom reasons
+
+When you declare a <<declaring_dependencies,dependency>> or a <<sec:dependency_constraints,dependency constraint>>, you can provide a custom reason for the declaration. This makes your build script and the dependency insight report easier to comprehend.
```

Comment:
I'd also rather say "easier to interpret".

Reply:
fixed